cmd/api-errors.go

```diff
@@ -506,6 +506,11 @@ var errorCodeResponse = map[APIErrorCode]APIError{
 		Description:    "Request is not valid yet",
 		HTTPStatusCode: http.StatusForbidden,
 	},
+	ErrSlowDown: {
```

Comment:
I think 'ServiceUnavailable' is better than SlowDown here.. isn't slow down returned to a specific user who is sending too many requests ?

Reply:
> I think 'ServiceUnavailable' is better than SlowDown here.. isn't slow down returned to a specific user who is sending too many requests ?

I withdraw what I said. We already have only one user.

Reply:
the reason for slowdown is AWS S3 error compliance @vadmeste the http status is correct and in accordance with `Retry-After` http header - https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Retry-After